### DISCUSSION TITLE
Zenodo DOI not updated from latest releases

### DISCUSSION THREAD

**Discussion by shikokuchuo** [ropensci/targets#1245] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

DOI still points to targets 1.4.0.

**Comment by wlandau** [comment]
Hmm... I set that up several years ago and trusted the process to be automatic. @maelle, do you happen to know if this is going on in other rOpenSci repos?

**Reply by shikokuchuo** [reply]
If that's the case, I remember they did a major upgrade of their system towards the end of last year. Perhaps that broke the integration.

**Comment by maelle** [comment]
Ouch, I find Zenodo so complicated!

On https://github.com/ropensci/targets/settings/hooks the Zenodo looks is indicated as "This hook has never been triggered" which makes no sense as it's supposed to be triggered by releases, and there are releases.

@wlandau do you by any chance see more information in the Zenodo record itself, as an admin? https://zenodo.org/records/10357140

@shikokuchuo I agree with your hypothesis.

**Reply by wlandau** [reply]
Hi @maelle, I logged in, but I seem to only be able to see versions of `targets` up through 1.4.0. 

![Screenshot 2024-02-28 at 8 44 50 AM](https://github.com/ropensci/targets/assets/1580860/f9555647-e7f4-48e3-accb-94a5330a8904)

**Reply by maelle** [reply]
do you have access to settings of some kind? where the GitHub integration is turned on?

**Reply by wlandau** [reply]
Ah, that would be it. Thanks for pointing me in the right direction. For some reason a bunch of ropensci repos were dropped. Added them again. Builds should show up eventually I think.

**Reply by maelle** [reply]
so weird. would you mind posting about this in the rOpenSci slack, in case others were bitten by this?

**Reply by wlandau** [reply]
It's odd, even after re-adding the repos that were dropped, I still see the latest `targets` release as 1.4.0. I posted to the `package-maintenance` channel referencing this thread.

**Reply by shikokuchuo** [reply]
I think it'll update when you make a new release.

**Reply by wlandau** [reply]
Looks like Zenodo found `targets` 1.6.0.

**Reply by maelle** [reply]
:tada: Zenodo is so hard. To succeed with the dev guide release on Zenodo, I enlisted @yabellini's help :grin:
